.gitignore

```diff
@@ -3,3 +3,4 @@ npm-debug.log
 yarn-error.log
 
 coverage/
+.vscode
```

Comment:
The project policy is to keep text editor ignores in local `.git/info/exclude` or in system git config `core.excludesFile`